src/frontend/src/components/base/Autocomplete/Autocomplete.tsx

```diff
@@ -65,47 +73,67 @@ export function Autocomplete({
             filteredSuggestions.filterText !==
                 filteredSuggestions.items[0].name);
 
+    useEffect(() => {
+        if (filteredSuggestions.filterText === "" && value && onClearCallback)
+            onClearCallback();
+    }, [filteredSuggestions.filterText]);
+
+    // Handles filter text set from other parent search,
+    // to keep the map search and list search values in sync
+    useEffect(() => {
+        if (value && filteredSuggestions.filterText !== value)
+            filteredSuggestions.setFilterText(value);
+    }, [value]);
```

Comment:
Wondering about the similar thing as mentioned above, but here for `filteredSuggestions.filterText` instead. Lemme know if I am missing something here.